dot-net/hcx-integrator-sdk/Io.HcxProtocol/Io - Backup.HcxProtocol.csproj

```diff
@@ -0,0 +1,107 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+
+		<RootNamespace>Io.HcxProtocol</RootNamespace>
+		<AssemblyName>Io.Hcx.Protocol</AssemblyName>
+		<Version>1.0.1</Version>
```

Comment:
version is 1.0.2?

Reply:
yes. Version is 1.0.2. Updated in the file.